Lesson3/Task3.java

```diff
@@ -0,0 +1,18 @@
+package Lesson3;
+
+public class Task3 {
+    public static void main(String[] args) {
+        int hourTime = 17;
+        String helloWords = "";
+        if (hourTime >= 5 && hourTime <12) {

```

Comment:
"Violation: There isn't separating white spaces for operators OR parameters
Solution: Add white spaces"